### PR TITLE
String de-duplication, consolidation, and cleanup

### PR DESCRIPTION
**Pull request by marcellamaki**
A great deal of string re-arranging

### CODE REVIEW

File: kolibri/core/assets/src/mixins/commonCoreStrings.js [hunk 1]
```diff
@@ -574,6 +583,10 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
     message: 'Explore',
     context: 'Resource and filter label for the type of learning activity. Translate as a VERB',
   },
+  activityType: {
+    message: 'Activity type',
+    context: 'Indicate the kind of type of learning activity',
```
Comment (radinamatic) [inline kolibri/core/assets/src/mixins/commonCoreStrings.js]: _kind of type_ sounds redundant... Maybe leave just the type and include examples: viewing, reading, etc.

File: packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue [hunk 2]
```diff
@@ -311,19 +311,15 @@
...
         message: 'Edit device sync schedule',
         context: 'Subtitle for the edit sync schedule page',
       },
       serverTime: {
         message: 'Server time:',
         context: 'Server time label',
       },
-      checboxlabel: {
+      checkboxlabel: {
         message: 'If scheduled sync fails, retry until the next scheduled sync',
```
Comment (radinamatic) [inline packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue]: Didn't we decide for this to be: `If scheduled sync fails, keep trying`?

File: packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue [hunk 3]
```diff
@@ -367,14 +355,6 @@
         message: 'Every Two Weeks',
```
Comment (radinamatic) [inline packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue]: Why is this title-capitalized? Same goes for `Every Week` above...

File: packages/kolibri-common/components/SyncSchedule/ManageSyncSchedule.vue [hunk 4]
```diff
@@ -268,13 +268,9 @@
     },
 
     $trs: {
-      toolbarHeader: {
+      syncSchedules: {
         message: 'Sync Schedules',
```
Comment (radinamatic) [inline packages/kolibri-common/components/SyncSchedule/ManageSyncSchedule.vue]: `Schedules` does not need to be capitalized.

File: packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue (hunk 2 above)
Comment (rtibbles) [reply packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue]: To go one further `checkboxLabel` to use camelCase?

File: kolibri/core/assets/src/views/sync/FacilityNameAndSyncStatus/index.vue [hunk 5]
```diff
@@ -60,10 +60,11 @@
 
   import { now } from 'kolibri.utils.serverClock';
   import taskStrings from 'kolibri.coreVue.mixins.commonTaskStrings';
+  import commonCoreStrings from '../../../mixins/commonCoreStrings';
```
Comment (rtibbles) [inline kolibri/core/assets/src/views/sync/FacilityNameAndSyncStatus/index.vue]: Preferable to use the coreAPI import path here, even though it's in core.

File: kolibri/plugins/facility/assets/src/views/CsvInfoModal.vue [hunk 6]
```diff
@@ -3,7 +3,7 @@
   <!-- eslint-disable max-len -->
   <KModal
     :title="$tr('csvDetails')"
-    :submitText="$tr('close')"
+    :submitText="coreString('close')"
```
Comment (rtibbles) [inline kolibri/plugins/facility/assets/src/views/CsvInfoModal.vue]: Which one is correct - `'close'` here or `'closeAction'` below?

File: kolibri/plugins/learn/assets/src/views/CompletionModal/index.vue [hunk 7]
```diff
@@ -130,8 +130,8 @@
             ref="closeButton"
             class="close-button"
             icon="close"
-            :ariaLabel="$tr('close')"
-            :tooltip="$tr('close')"
+            :ariaLabel="coreString('close')"
```
Comment (rtibbles) [inline kolibri/plugins/learn/assets/src/views/CompletionModal/index.vue]: close or closeAction again

File: packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue [hunk 8]
```diff
@@ -71,7 +71,7 @@
 
           <p class="spacing">
             <KCheckbox>
-              {{ $tr('checboxlabel') }}
+              {{ $tr('checkboxlabel') }}
```
Comment (rtibbles) [inline packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue]: I flagged elsewhere that this should probably be `'checkboxLabel'` (both here and where it's defined).

File: packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue [hunk 9]
```diff
@@ -311,20 +311,16 @@
...
         message: 'Edit device sync schedule',
         context: 'Subtitle for the edit sync schedule page',
       },
       serverTime: {
         message: 'Server time:',
         context: 'Server time label',
       },
-      checboxlabel: {
-        message: 'If scheduled sync fails, retry until the next scheduled sync',
+      checkboxlabel: {
```
Comment (rtibbles) [inline packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue]: checkboxLabel

**Review by rtibbles — COMMENTED** [review]
A couple more things caught my eye.

**Review by radinamatic — APPROVED** [review]
So much closer, can't wait for @pcenov to test the assets from this, so we can confirm all is good and push the final version on Crowdin! 💯 ⏳ :shipit:

**Review by rtibbles — APPROVED** [review]

**Pull request merged**